tasks/main.yml

```diff
@@ -10,51 +10,32 @@
   stat: path=/usr/bin/gitlab-ctl
   register: gitlab_file
 
-# Install GitLab and its dependencies (RedHat).
-- name: Install GitLab dependencies (RedHat).
-  yum: "name={{ item }} state=installed"
+# Install GitLab and its dependencies.
+- name: Install GitLab dependencies.
```

Comment:
Actually it is nice that the dependencies have the same name in both OS's families, have the same name for the dependency packages.

    - openssh-server		
    - postfix		
    - curl

If you prefer, I will leave the "(RedHat)", but I will prefer to declare the requisites as a list under `vars/{{ ansible_os_family }}` and install them this way.

    - name: Install GitLab dependencies (Debian).		
      package: "name={{ item }} state=installed"		
      with_items: "{{ gitlab_requisites }}"

If you agree, I will make another PR for this change.